priv/repo/migrations/20210702185710_migrate_notification_events.exs

```diff
@@ -0,0 +1,115 @@
+defmodule Console.Repo.Migrations.MigrateNotificationEvents do
+  use Ecto.Migration
+  alias Console.Channels
+
+  def cast_label_notification_event(record) do
+    %{
+      id: Enum.at(record, 0),
+      reported_at: Enum.at(record, 1),
+      key: Enum.at(record, 2),
+      label_id: Enum.at(record, 3),
+      details: Enum.at(record, 4),
+      sent: Enum.at(record, 5),
+      inserted_at: Enum.at(record, 6),
+      updated_at: Enum.at(record, 7)
+    }
+  end
+
+  def up do
+    result = Ecto.Adapters.SQL.query!(Console.Repo, """
+      SELECT id, reported_at, key, label_id, details, sent, inserted_at, updated_at FROM label_notification_events;
+    """).rows
+    
+    for i <- result do
+      old_row = cast_label_notification_event(i)
+      node_id = cond do
+        old_row.key in ["device_deleted", "device_join_otaa_first_time", "device_stops_transmitting", "downlink_unsuccessful"] ->
+          old_row.label_id
+        true ->
+          channel = Channels.get_channel(old_row.details["channel_id"])
```

Comment:
for this to work properly, we need this merged to `staging`: https://github.com/helium/console/pull/676